yb-voyager/cmd/cutover.go

```diff
@@ -44,6 +45,7 @@ func InitiatePrimarySwitch(action string) error {
 	if err != nil {
 		return err
 	}
+	log.Infof("cutover initiated, wait for it to complete")
```

Comment:
Did you intend to show this message to user? If yes, then `utils.PrintAndLog()`?